test/run_test.py

```diff
@@ -390,6 +390,21 @@
     'test_jit_fuser_legacy',
 ]
 
+# Dictionary matching test modules (in TESTS) to lists of test cases (within that test_module) that would be run when
+# options.run_specified_test_cases is enabled.
+# For example:
+# {
+#   test_nn -> ([test_doubletensor_avg_pool3d, test_share_memory, test_hook_requires_grad]),
+#   ...
+# }
+# For test_nn.py, we would ONLY run test_doubletensor_avg_pool3d, test_share_memory, and test_hook_requires_grad.
+SPECIFIED_TEST_CASES_DICT: Dict[str, List[str]] = {}
```

Comment:
I dont think we need this as global.
can we do something similar to the `bring_to_front` option.

Reply:
Sorry, I'm not exactly sure what you mean by something similar to the bring_to_front option. Do you mean not having this as global and instead passing it through functions (similar to what is currently being done with slow tests)?

Reply:
This would come at the cost of adding a parameter to a lot of extra functions (like all the CUSTOM_HANDLERS) and run_test_module, run_test, and get_test_case_args. Would that be worth it?